configs/components/runtime-pe-installer.rb

```diff
@@ -0,0 +1,28 @@
+# This component exists to link in the gcc runtime libraries.
+component "runtime-pe-installer" do |pkg, settings, platform|
```

Comment:
Missing `c` here

Reply:
I'm confused; where is the missing `c`?

Reply:
Hmm, I'm seeing `omponent` instead of `component` in the github ui, but I see `component` the email I got with your comment :woman_shrugging:

Reply:
Huh, must be a ghost commit that I force pushed over. 👻